contributions/demo/week6-software-bots/khalidey-vishaln/README.md

```diff
@@ -0,0 +1,25 @@
+# Assignment Proposal
+
+## Title
+
+Remote Monitoring and controlling of Machine Learning Models with Telegram Bot
+
+## Names and KTH ID
+
+- Khalid El Yaacoub (khalidey@kth.se)
+- Vishal Nedungadi (vishaln@kth.se)
+
+## Deadline
+
+Deadline for task 4
+
+## Category
+
+Demo
+
+## Description
+
+Software bots is a very fascinating demo. Did you know that there were bots for Machine Learning as well? We are not talking about bots that use Machine Learning.
```

Comment:
first sentence doesn't read well

Reply:
@cesarsotovalero 
We have updated the Description to make it more clear. Let us know what you think.